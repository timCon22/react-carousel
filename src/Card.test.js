import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";


it("renders withoug crashing", function(){
  render(<Card/>)
})


it("matches snapshot", function() {
    const { asFragment } = render(<Card caption="Photo by Richard Pasquarella on Unsplash"/>);
    expect(asFragment()).toMatchSnapshot();
});
  

it("matches snapshot", function() {
    const { asFragment } = render(<Card caption="Photo by Pratik Patel on Unsplash" />);
    expect(asFragment()).toMatchSnapshot();
});


it("matches snapshot", function() {
    const { asFragment } = render(<Card caption="Photo by Josh Post on Unsplash" />);
    expect(asFragment()).toMatchSnapshot();
});