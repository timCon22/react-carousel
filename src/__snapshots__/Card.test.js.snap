// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Photo by Richard Pasquarella on Unsplash
    </h4>
    <img
      alt="Photo by Richard Pasquarella on Unsplash"
      class="Card-image"
    />
    <small
      class="Card-small"
    >
      Image  of .
    </small>
  </div>
</DocumentFragment>
`;

exports[`matches snapshot 2`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Photo by Pratik Patel on Unsplash
    </h4>
    <img
      alt="Photo by Pratik Patel on Unsplash"
      class="Card-image"
    />
    <small
      class="Card-small"
    >
      Image  of .
    </small>
  </div>
</DocumentFragment>
`;

exports[`matches snapshot 3`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Photo by Josh Post on Unsplash
    </h4>
    <img
      alt="Photo by Josh Post on Unsplash"
      class="Card-image"
    />
    <small
      class="Card-small"
    >
      Image  of .
    </small>
  </div>
</DocumentFragment>
`;
